syncer/syncer.go

```diff
@@ -1295,7 +1286,10 @@ func (s *Syncer) syncDML(
 			t := v.(int)
 			time.Sleep(time.Duration(t) * time.Second)
 		})
-		return db.ExecuteSQL(tctx, queries, args...)
+		// use background context to execute sqls as much as possible
+		ctctx, cancel := tctx.WithContext(context.Background()).WithTimeout(maxDMLConnectionDuration)
```

Comment:
Seems can use tctx.WithTimeout(maxDMLConnectionDuration)

Reply:
Want to make sure this context is not affected by the parent context.

Reply:
I think this context with an timeout durtion of `5m` is not acceptable. In the worst case, the worker need to wait 5m before exit.